Makefile

```diff
@@ -290,8 +291,11 @@ $(BUILD)/protoc: $(PROTO_FILES) $(BIN)/$(PROTOC_VERSION_BIN) $(BIN)/protoc-gen-g
 		--yarpc-go_out=$(PROTO_OUT) \
 		$$(find $(PROTO_DIR) -name '*.proto');\
 	)
-	$Q cp -R $(PROTO_OUT)/uber/cadence/* $(PROTO_OUT)/
-	$Q rm -r $(PROTO_OUT)/uber
+	$Q # This directory exists for local/buildkite but not for docker builds.
```

Comment:
hmm.  should this be running for docker builds actually?  we should probably not re-generate the committed files when publishing - better to publish a known-good state.